app/views/organization_account_requests/new.html.erb

```diff
@@ -1,92 +1,106 @@
 <section class="pt-5 pb-5">
-  <div class="container mb-md-5 mt-md-5 ">
-    <div class="container mb-md-5 mt-md-5">
-      <div class="text-center mb-5">
-        <h1 class="section-heading text-capitalize underline">
-          <%= t('.header') %>
-        </h1>
-      </div>
+  <div class="container mb-md-5 mt-md-5">
+    <div class="text-center mb-5">
+      <h1 class="section-heading text-capitalize underline">
+        <%= t('.header') %>
+      </h1>
     </div>
 
     <div class="row">
-      <div class='col-md-6 mb-3 text-center bigger mx-auto'>
+      <div class="col-md-6 mb-3 text-center bigger mx-auto">
         <%= t('.description') %>
       </div>
     </div>
+
     <div class="row">
       <div class="col-md-6 mx-auto card p-5">
-      <%= bootstrap_form_with model: OrganizationAccountRequest.new, url: organization_account_request_path do |form| %>
-        <% if @organization_acccount_request.errors.count > 0 %>
-          <div class="alert alert-danger mt-1" role="alert">
-            <%= t('organizations.staff.organizations.form.please_fix_the_errors') %>
-          </div>
-          <ul>
-            <% @organization_acccount_request.errors.full_messages.each do |message| %>
-              <div class="alert alert-danger mt-1" role="alert">
-                <%= message %>
-              </div>
-            <% end %>
-          </ul>
-        <% end %>
-
-        <div class='card-body'>
-          <div class="row mt-3">
-            <div class="col-lg-6">
-              <form class="mb-3">
+        <%= bootstrap_form_with model: OrganizationAccountRequest.new, url: organization_account_request_path do |form| %>
+          <% if @organization_account_request.errors.any? %>
+            <div class="alert alert-danger mt-1" role="alert">
+              <%= t('organizations.staff.organizations.form.please_fix_the_errors') %>
+            </div>
+            <ul>
+              <% @organization_account_request.errors.full_messages.each do |message| %>
+                <li><%= message %></li>
+              <% end %>
+            </ul>
+          <% end %>
 
+          <div class="card-body pt-0">
+            <div class="row mt-3">
+              <div class="col-lg-6">
                 <div class="form-group">
                   <%= form.text_field :name,
-                    label: 'Organization name',
-                    class: 'form-control',
-                    required: true
-                   %>
+                                      label: t('organization_account_requests.form.organization_name'),
```

Comment:
I prefer using/seeing the "Lazy" look up. I care less about it being shorter but more importantly is forces a consistent organizational structure in the en.yml file. 

This example is only a small change to the translation file but I think contributors tend to look at the code we have and try to emulate it. The translation file could get out of hand pretty quickly if people make up a different look up path for each pr.

```suggestion
                                      label: t('.form.organization_name'),
```

Reply:
Ah yes good call. I forgot about the lazy lookup. I will implement that.